scale/job/serializers.py

```diff
@@ -198,6 +198,21 @@ class JobExecutionBaseSerializer(ModelIdSerializer):
     error = ModelIdSerializer()
 
 
+class JobExecutionBaseSerializer(ModelIdSerializer):
+    """Converts job execution model fields to REST output"""
+    status = serializers.CharField(source='get_status')
+    timeout = serializers.IntegerField()
+
+    created = serializers.DateTimeField()
+    queued = serializers.DateTimeField()
+    started = serializers.DateTimeField()
+    ended = serializers.DateTimeField(source='jobexecutionend.ended')
+
+    job = ModelIdSerializer()
+    node = ModelIdSerializer()
+    error = ModelIdSerializer()
```

Comment:
Add job_type = ModelIdSerializer() to base serializer
Move timeout from JobExecutionBaseSerializer to JobExecutionSerializer
Move exe_num and cluster_id from JobExecutionSerializer to JobExecutionBaseSerializer
Update docs appropriately